code/game/objects/items/devices/megaphone.dm

```diff
@@ -8,15 +8,19 @@
 	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
 	w_class = WEIGHT_CLASS_SMALL
 	siemens_coefficient = 1
-	var/spamcheck = 0
+	var/last_used = 0
 	var/list/voicespan = list(SPAN_COMMAND)
 
 /obj/item/megaphone/suicide_act(mob/living/carbon/user)
 	user.visible_message("<span class='suicide'>[user] is uttering [user.p_their()] last words into \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
-	spamcheck = 0//so they dont have to worry about recharging
+	last_used = 0//so they dont have to worry about recharging
```

Comment:
does this even work in the first place I'd think it would set the recharge time on the say directly after this

Reply:
it's used to make sure the megaphone would work ~~but it'd be funnier for a suicide shame if it wasnt charged~~